tms-grpc-service/src/utils/logger.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console, @typescript-eslint/keyword-spacing */
```

Comment:
Please implement ELK logger. Check this file in this PR for advice. https://github.com/mojaloop/fraud_risk_management/pull/190/files#diff-9055b38a06a7a164422eab2265687195e7fa0cfb3672bf8b7fd2614328e06434